client/package.json

```diff
@@ -1,58 +1,92 @@
 {
   "name": "@azure/event-hubs",
-  "version": "1.0.8",
+  "version": "2.0.0-preview.1",
   "description": "Azure Event Hubs SDK for JS.",
   "author": "Microsoft Corporation",
   "license": "MIT",
+  "homepage": "http://github.com/azure/azure-sdk-for-js",
+  "repository": "github:Azure/azure-sdk-for-js",
```

Comment:
Let's not add links (homepage, repository, and url) to the new sdk until we move the project.
This is to ensure that if we had to ship a release from this repo before the move, we can still do it

Reply:
Agree. I will revert the EPH change too and track the update change in a new issue.